packages/main/src/plugin/dialog-registry.spec.ts

```diff
@@ -81,12 +82,19 @@ const tmpMyPath = path.resolve(tmpdir(), 'my/path');
 
 describe('showOpenDialog', () => {
   beforeEach(() => {
+    vi.resetAllMocks();
     vi.mocked(dialog.showOpenDialog).mockResolvedValue({
       filePaths: [tmpMyPath],
       canceled: false,
     });
   });
 
+  vi.mock('../util.js', () => {
```

Comment:
```suggestion
  vi.mock(import('../util.js'), () => {
```

suggest: to make it fail if for some reason we move util file to another location (in one case it'll fail the import, in the second case it will be silently ignored)